index.js

```diff
@@ -23,9 +24,13 @@ module.exports = (function () {
         wait = true;
     };
 
+    lazyLoad.setCLIOptions = function (options) {
+        if (!options) { options = {}; }
+        cliOptions = options;
+    };
```

Comment:
i think we should avoid statefulness on the main export. instead, could we have an option passed to `tape.run`, that passes it in turn to `getHarness`?

Reply:
okay will do that

Reply:
@ljharb, so while trying to do this, I realize the `harness` is already defined by the time the `tape.run` is called, hence `getHarness` returns the already created harness which does not have the needed config passed to it.

Kindly let me know if there's something I'm missing. Thanks

Reply:
Hmm, I’ll have to think more about this.

We could make the harness look up the value from the env always, but then we’d have to set it in bin/tape.

Reply:
> We could make the harness look up the value from the env always, but then we’d have to set it in bin/tape.

okay, based on [your comment](https://github.com/substack/tape/pull/572#pullrequestreview-860245563) here, this approach could help achieve that

Reply:
> We could make the harness look up the value from the env always, but then we’d have to set it in bin/tape.

@ljharb  so i've implemented the above suggestion and updated the pr, kindly do let me know if there are no crimes committed while doing so :smile:

Reply:
just out of curiosity and to help improve, maybe one thing left lingering would be to appreciate why statefulness on the main export would be good to avoid as highlighted [here](https://github.com/substack/tape/pull/572#discussion_r790189739). if that's okay to ask